skin/src/main/res/values/strings.xml

```diff
@@ -113,7 +113,7 @@
     <string name="rss_settings_general_software_notices">Software Notices</string>
     <string name="rss_settings_general_leave_study">Leave Study</string>
     <string name="rss_settings_general_join_study">Join Study</string>
-    <string name="rss_settings_version">Build number %1$s (%2$s)</string>
+    <string name="rss_settings_version">Build number %1$s (%2$d)</string>
```

Comment:
For some reason this just started showing as an error for me even though it hasn't changed.  Version code is an integer and not a string though so this is correct.